torch/nn/modules/loss.py

```diff
@@ -1160,20 +1163,22 @@ class probabilities only when a single class label per minibatch item is too res
         >>> output = loss(input, target)
         >>> output.backward()
     """
-    __constants__ = ['ignore_index', 'reduction', 'label_smoothing']
+    __constants__ = ['ignore_index', 'reduction', 'label_smoothing', 'dim']
     ignore_index: int
     label_smoothing: float
+    dim: int
 
     def __init__(self, weight: Optional[Tensor] = None, size_average=None, ignore_index: int = -100,
-                 reduce=None, reduction: str = 'mean', label_smoothing: float = 0.0) -> None:
+                 reduce=None, reduction: str = 'mean', label_smoothing: float = 0.0, dim: int = None) -> None:
```

Comment:
as indicated by the linter, the type here should be `dim: Optional[int]`

Reply:
@jbschlosser, I've corrected this.